### DISCUSSION TITLE
steam error: a generic faliure from the steamworks API

### DISCUSSION THREAD

**Discussion by vgiamp10** [WilliamVenner/gmpublisher#137] [Help]
![image](https://user-images.githubusercontent.com/84873533/193641060-9efd5cde-efda-445e-b1d5-61a2da540756.png)
idk why it gives me this error

**Comment by Chickzz** [comment]
i have the same error,
i wanted to upload a addon but it shows this instead

**Comment by Chickzz** [comment]
(update)
i changed my logged in Steam account to the one that actually owns the game and can make Post on the Community Workshop and it worked!
